charmcraft.yaml

```diff
@@ -16,13 +16,17 @@ bases:
     - name: ubuntu
       channel: "22.04"
 
-# config:
-#   options:
-#     snap_channel:
-#       default: "latest/stable"
-#       type: string
-#       description: |
-#           If install_method is set to "snap" this option controls channel name.
+requires:
+  credentials:
+    interface: keystone-admin
+
+config:
+  options:
+    ssl_ca:
+      default:
+      type: string
+      description: |
+        Custom SSL CA for keystone if required.
```

Comment:
Sorry, one final spot: we should probably document this a bit more (it's a base64 encoded string or plain multi-line string (because it's a CA)). Users coming from the old [charm](https://git.launchpad.net/charm-prometheus-openstack-exporter/tree/src/reactive/openstack_exporter.py#n192) will assume that it's base64 encoded. For me, multi-line string is kind of hard to use

Reply:
Sure, that's fine. Just want to point that out loudly